matrix.md

```diff
@@ -0,0 +1,136 @@
+# Compartmentalization Evaluation Matrix
+
+> Complete this matrix for a single implementation or system.  
+> Use ✅ / ❌ for checkboxes, categorical scales, and short text values as indicated.
+
+---
+
+## Security
+
+| **Dimension** | **Metric** | **Value / Notes** |
+|----------------|------------|-------------------|
+| **Isolation Model** | | |
+| Primary use case | Untrusted library isolation / Secret/key protection / Fault isolation / Supply chain defense / Other: ___ | |
+| Subject selection | Code-centric / Data-centric / Hybrid | |
+| Finest isolation granularity | Function / Library / Thread / Process / VM / Other: ___ | |
+| **Isolation Approach** | | |
+| Hardware primitive(s) | MPK/PKU / MTE / CHERI / TEEs / Other: ___ / None | |
```

Comment:
Similar comment here on the two different types of "/". Perhaps you can use parentheses (e.g., PKU, PKS).